src/videojs.ima.js

```diff
@@ -1092,7 +1092,22 @@
       clearInterval(updateTimeIntervalHandle);
       clearInterval(seekCheckIntervalHandle);
       clearInterval(resizeCheckIntervalHandle);
-      player.one('play', player.ima.setUpPlayerIntervals_);
+      if(player.el()) player.one('play', player.ima.setUpPlayerIntervals_);
```

Comment:
I think we you can revert this - I'll update the docs to indicate that you need to create the plugin before calling play on the video element.